table/column_test.go

```diff
@@ -104,12 +104,16 @@ func (t *testTableSuite) TestCheck(c *C) {
 	col := newCol("a")
 	col.Flag = mysql.AutoIncrementFlag
 	cols := []*Column{col, col}
-	CheckOnce(cols)
+	err := CheckOnce(cols)
+	c.Assert(err, NotNil)
```

Comment:
please check whether the error message is expected, not only check whether is an error message.

Reply:
Added err msg check.